### DISCUSSION TITLE
FP v. 1.11.3 Preview pre-release is generating some error messages

### DISCUSSION THREAD

**Discussion by spiritusanimalis** [freeplane/freeplane#1165] [Help]
Hello everyone,
I'm getting some error messages in FP logs. One in particular concerns the XML version being used. I've attached the log.0 log file in txt format.
[log.0.txt](https://github.com/freeplane/freeplane/files/11536393/log.0.txt)

**Comment by spiritusanimalis** [comment]
Edit: the XML error is in the current stable release too.

**Reply by dpolivaev** [reply]
I do not see any xml related error messages.
It complains only about two missing or bad user icon files:

SEVERE: can not load icon '/images/freeIBIS-icon.svg'
SEVERE: can not load icon '/images/userIcons-icon.svg'

Both files seem to be a part of add-on or otherwise installed by the user.
